packages/storage-plugin/src/storage.module.ts

```diff
@@ -31,12 +32,16 @@ export class NgxsStoragePluginModule {
         {
           provide: NGXS_STORAGE_PLUGIN_OPTIONS,
           useFactory: storageOptionsFactory,
-          deps: [USER_OPTIONS]
+          deps: [USER_OPTIONS, DEFAULT_STATE_KEY]
         },
         {
           provide: STORAGE_ENGINE,
           useFactory: engineFactory,
           deps: [NGXS_STORAGE_PLUGIN_OPTIONS, PLATFORM_ID]
+        },
+        {
+          provide: DEFAULT_STATE_KEY,
+          useValue: (options && options.defaultStateKey) || '@@STATE'
```

Comment:
PS. I see that you are using an InjectionToken and a provider to communicate this value internally.
Is there any reason to do this over just having the additional property in the options?
Does this give any additional locus of control for MFE's?

Reply:
There's no particular reason for this. I was doing a rapid programming 🤣